bitseq/sequence.go

```diff
@@ -273,12 +272,9 @@ func (h *Handle) set(ordinal uint32, any bool, release bool) (uint32, error) {
 			continue
 		}
 
-		// Unless unexpected error, save private copy to local copy
+		// Previous atomic push was succesfull. Save private copy to local copy
 		h.Lock()
 		defer h.Unlock()
-		if h.dbIndex != ci {
-			return ret, fmt.Errorf("unexected database index change")
-		}
```

Comment:
Thanks. That makes me recollect the if check was meant to be against nh.Index (it was a paranoid check)

Right, there should be no need to copy the dbIndex, given h will get it updated by the watch